packages/react-accordion/package.json

```diff
@@ -13,6 +13,7 @@
   "license": "MIT",
   "scripts": {
     "build": "just-scripts build",
+    "bundle-size": "bundle-size measure",
```

Comment:
is this command necessary if it's already a bin in the root ? I guess it might be for lage ?

Reply:
> is this command necessary if it's already a bin in the root ? I guess it might be for lage ?

Exactly, as we run commands per package to allow scoping and `lage` needs to know what to run.